uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+
+<launch>
+    <!-- TODO(wraftus): Make names of ZEDs more descriptive depending on location on robot-->
+    <arg name="zed_id1"               default="0" />
+    <arg name="verbose"               default="false" />
+
+    <arg name="resolution"            default="2" /> 
+    <!--0=RESOLUTION_HD2K, 1=RESOLUTION_HD1080, 2=RESOLUTION_HD720, 3=RESOLUTION_VGA -->
+    <arg name="frame_rate"            default="30" />
+    <!-- RESOLUTION_HD2K   -> 2208*1242, available framerates: 15 fps.
+         RESOLUTION_HD1080 -> 1920*1080, available framerates: 15, 30 fps.
+         RESOLUTION_HD720  -> 1280*720,  available framerates: 15, 30, 60 fps.
+         RESOLUTION_VGA    -> 672*376,   available framerates: 15, 30, 60, 100 fps. -->
+
+    <!-- Definition coordinate frames -->
+    <arg name="publish_tf"                    default="true" />
+    <arg name="publish_map_tf"                default="true" />
+    <arg name="base_frame"                    default="zed_camera_center" />
+
+    <arg name="depth_topic"                   default="depth_registered" />
```

Comment:
thought I'd keep them out here so if we  want to change it we don't have to do it for all zeds & depthimage_to_laserscans

Reply:
Since the chances of us changing them are so low, I would prefer a cleaner interface (Less options to confuse you when `roslaunch`ing) than functionality we're not going to use.